packages/components/src/truncate/README.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
Does it require any additional tooling to regenerate README file?

Reply:
It seems like there is some different tools used to generate that README file. I removed the extra markup and now it looks like the rest of component's README's.